proposed/project-file-simplification.md

```diff
@@ -0,0 +1,351 @@
+# Project file simplification and improved change management
+
+// Contents of Type should match template name as much as possible
+
+## Summary
+[summary]: #summary
+
+This proposal is to simplify the project file while giving the user better control over breaking changes and new warnings.
+
+The approach is versioned default properties specified by a `Base` attribute in the project file, a `Directory.Build.props`, or an `Import` file. This attribute specifies a version of the file containing the default properties. The project file is further simplified by `Type` attribute on the `Project` element. The `Type` is not versioned, although versioned default properties can be added as part of the `Base` evaluation. The `Type` attribute replaces the current `SDK` and the `Type` file contains both the `SDK` name and properties that apply to the application `Type`.
+
+An example of a new project file is:
+
+```XML
+<Project Base="10.0.100" 
+         Type="Console"> 
+</Project> 
+```
+
+If there is a `Directory.Build.props` or `Import` file, it can contain the `Base` attribute, although it cannot contain the `Type` attribute. This is because the `Type` attribute specifies the SDK instructions for build evaluation.
+
+There would be no change to existing projects. They will continue to work as today.
+
+## Motivation
+[motivation]: #motivation
+
+The fundamental benefit of this effort is to speed up and remove friction for users that wish to adopt tooling improvements quickly, while also allowing users to delay incorporating change until it is not disruptive. To do this, we need to allow developers a friction free experience on upgrade, which means that with no more work than previously they have control over the upgrade process.
```

Comment:
What does friction free mean here though? There are always behavior differences when updating between major versions of the .NET SDK. Are we saying that all behaviors are tied to `Base`, just diagnostics, etc ... Need to be clear where the line is with this.